templates/python-poetry/CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+We are happy if you want to contribute to this project. If you find any bugs or have suggestions for improvements, please open an issue. We are also happy to accept your PRs. Just open an issue beforehand and let us know what you want to do and why.
+
+## Development setup
+
+1. Clone the repository and navigate inside the new directory.
+
+2. Install dependencies using [Poetry](https://python-poetry.org)
+
+```sh
+poetry install
+```
+
+3. Install the pre-commit hooks for linting and to enforce a consistent code style.
+
+```sh
+pre-commit install
+```
```

Comment:
I think these steps don't apply for this repo 😅

Reply:
yes, it doesn't apply to this repository. it is not linked in the root readme, only for the Python project. A similar one should be created for the Java template (#21)

Reply:
Ah, my bad, I overlooked that this file is not in root.